src/test/java/com/rabbitmq/integration/tests/DelayedAMQPQueueMessageIT.java

```diff
@@ -0,0 +1,69 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2013-2020 VMware, Inc. or its affiliates. All rights reserved.
+package com.rabbitmq.integration.tests;
+
+import com.rabbitmq.TestUtils;
+import com.rabbitmq.jms.admin.RMQDestination;
+import com.rabbitmq.jms.util.Shell;
+import jakarta.jms.*;
+import org.junit.jupiter.api.*;
+
+import java.io.IOException;
+import java.util.concurrent.CountDownLatch;
+import java.util.stream.IntStream;
+
+import static com.rabbitmq.Assertions.assertThat;
+import static com.rabbitmq.TestUtils.onCompletion;
+import static java.lang.String.valueOf;
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Integration test
+ */
+public class DelayedAMQPQueueMessageIT extends AbstractAmqpITQueue {
+
+    String queueName = "DelayedAMQPQueueMessageIT";
+    RMQDestination destination;
+
+    @BeforeEach
+    void init() throws Exception {
+        destination = new RMQDestination(queueName, true, false);
+    }
+
+    @BeforeEach
+    public void beforeMethod() throws IOException {
+        try {
```

Comment:
Please use a JUnit condition like `DisabledIfTlsNotEnabled` in `SSLHostnameVerificationIT`. It uses `TestUtils#tlsAvailable()` under the covers, which just checks the content of the output of `rabbitmqctl status`. Something on the same model to check the plugin activation should work and it would also remove the need for `rabbitmq-plugins` and so another system property in the build.